api.planx.uk/modules/send/bops/bops.test.ts

```diff
@@ -37,7 +37,7 @@ describe(`sending an application to BOPS (v2)`, () => {
         bopsApplications: [],
       },
       variables: {
-        session_id: "123",
+        session_id: "37e3649b-4854-4249-a5c3-7937c1b952b9",
```

Comment:
API tests now require that a `session_id` is an actual UUID, not just a random string.